crates/bevy_window/src/event.rs

```diff
@@ -96,3 +98,18 @@ pub struct WindowMoved {
     pub id: WindowId,
     pub position: IVec2,
 }
+
+/// Updates the CursorPosition resource with the latest CursorMoved events
+pub fn cursor_movement_res_system(
+    mut cursor_position: ResMut<CursorPosition>,
+    mut cursor_position_events: EventReader<CursorMoved>,
+    mut window_close_events: EventReader<WindowCloseRequested>,
+) {
+    for event in cursor_position_events.iter() {
+        cursor_position.positions.insert(event.id, event.position);
+    }
+
+    for event in window_close_events.iter() {
+        cursor_position.positions.remove(&event.id);
+    }
+}
```

Comment:
What happens when we get a `CursorLeft` event?
Is it worth removing the cursor position in that case as well?
(This may be incorrect, however I just wanted to bring this up just in case).

Reply:
It makes sense to me actually, I added that as well.

(it's not like it would cause any harm if it wasn't cleared from that, but it feels cleaner that it is)